pkg/kubelet/server/stats/volume_stat_calculator_test.go

```diff
@@ -263,6 +267,13 @@ func expectedFSStats() kubestats.FsStats {
 	}
 }
 
+func expectedVolumeHealthStats() *kubestats.VolumeHealthStats {
+	metric := expectedMetrics()
```

Comment:
shouldn't you have the nil check here too?

Reply:
no. expectedMetrics is a function that is only used in test: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/server/stats/volume_stat_calculator_test.go#L235

Reply:
Even if it is a test, I think we should still check nil to be safe.